compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/scene/DesktopComposeSceneLayer.desktop.kt

```diff
@@ -73,7 +74,10 @@ internal abstract class DesktopComposeSceneLayer(
      */
     private var maxDrawInflate = IntRect.Zero
 
-    private var outsidePointerCallback: ((eventType: PointerEventType) -> Unit)? = null
+    private var outsidePointerCallback: ((
+        eventType: PointerEventType,
+        button: PointerButton?
+    ) -> Unit)? = null
```

Comment:
Maybe worthwhile to typealias it.

Reply:
Or some event type with reduced scope as parameter. But the problem here - it should be public